src/db/config/config.ts

```diff
@@ -0,0 +1,30 @@
+require('dotenv').config();
+
+const config = {
+  development: {
+    username: process.env.DB_USER,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    dialect: 'postgres',
+    logging: false,
+  },
+
+  test: {
+    // dialect: 'postgres',
+    // enter test database credentials here
+  },
+
+  staging: {
+    // dialect: 'postgres',
+    // enter staging database credentials here
+  },
+
+  production: {
```

Comment:
I'm not sure it is within the scope of your task to add configuration for other environments.

Reply:
As you can see it's commented.
It serves as a guide for whoever is adding those db credentials, `// dialect: 'postgres',` is compulsory else it won't work.

Reply:
As you can see it's commented.
It serves as a guide for whoever is adding those db credentials, `// dialect: 'postgres',` is compulsory else it won't work.